packages/react/src/search/search.react.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Meta, omit, Story, storyOf } from '../../../../docs';
+import { Search, SearchProps } from './search.react';
+
+export default {
+  title: 'React/Search',
+  component: Search,
+  argTypes: {
+    ...omit<HTMLInputElement>('type'),
```

Comment:
Why not here:

```suggestion
    ...omit<SearchProps>('className', 'style'),
```

Curious 🤔

Reply:
Well `omit<SearchProps>('className', 'style')` doesn't do anything, neither prop is included in the table.

Then, `type` is `Omit`ted from `InputProps` in `SearchProps`, but it was showing up in the table for some reason 🤔 
Just tried removing it and now it works, so I'll remove this line.